desktop/appcontainer/ToastSource.ts

```diff
@@ -94,8 +96,36 @@ class ToastSourceLocalModel extends HoistModel {
         // We want to just memoize this by two args (one object)?  Is there a library for this?
         const toasters = toasterMap.get(container) || {};
         if (!toasters[position])
-            toasters[position] = OverlayToaster.create({position, className}, container);
+            toasters[position] = await this.createToaster({position, className}, container);
         toasterMap.set(container, toasters);
         return toasters[position];
     }
+
+    /** Workaround to avoid calling OverlayToaster.create(). It uses ReactDOM.render
+     * that gives a warning because it is deprecated in React 18.
+
+     * Is currently set to be removed in Blueprint v6.0
```

Comment:
what is set to be removed?  The create method?  Is this code from Blueprint v6?

Reply:
Unless we are explicitly trying not to touch this new `createToaster` code,  the signature can be simplified -- it always takes props, and an element, and returns a Promise<OverlayToaster>.

Reply:
So react deprecated the ReactDOM.render and current blueprint v5 toaster.create uses ReactDOM.render.(the reason we are getting the warning) They have notes that say they plan to remove the use of ReactDOM.render in v6.
https://github.com/palantir/blueprint/issues/5212#issuecomment-1294958195